### PR TITLE
Feat: Set up Docusaurus + Adds Radio Tokens doc

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- [x] Set docusaurus config to work with .scss and import global tokens
- [x] Adds Tokens Component (`TokenTable`, `TokenRow`, `TokenDivider`)
- [x] Updates Radio doc page

## How it works?

Adds design tokens and updates Radio documentation template on '/docs'

## How to test it?

See the [preview](https://faststore-git-chore-radio-doc-set-up-faststore.vercel.app/reference/ui/atoms/Radio) and check if any important information is missing for using this component.

### CODE REVIEW

File: apps/docs/docs/reference/ui/atoms/Radio.mdx [hunk 1]
```diff
@@ -16,10 +18,16 @@ Radios allow users to select one option from a set.
...
+```
+
 ## Usage
 
 ```tsx live
-<Radio/>
+<Radio />
 ```
 
 ## Props
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/atoms/Radio.mdx:33]: Still need to work on that! 
Props are not sync with `faststore/components`

image

File: apps/docs/docs/reference/ui/atoms/Radio.mdx [hunk 2]
```diff
@@ -1,31 +1,169 @@
...
 
 <PropsSection name="Radio" />
 
+## Design Tokens
```
Comment (renatamottam) [inline apps/docs/docs/reference/ui/atoms/Radio.mdx:49]: These `#h1`, `##h2`, `###h3` and `####h4` titles are all the same size 🥺 , can we fix it?

**Review by renatamottam — APPROVED** [review]
Really good approach, fan!

File: apps/docs/docs/reference/ui/atoms/Radio.mdx (hunk 2 above)
Comment (hellofanny) [reply apps/docs/docs/reference/ui/atoms/Radio.mdx:49]: Fixed!

**Comment by hellofanny** [comment]
@carolinamenezes  hey carol!  I'll merge this PR. But please review it, and feel free to get in touch with suggestions or anything else <3

**Pull request merged**